### PR TITLE
Bring in the latest from https://github.com/heroku/heroku-buildpack-g…

### PR DESCRIPTION
**Pull request by marybeshaw**
… https://github.com/heroku/heroku-buildpack-github-netrc while keeping our special "https" GitHub config at the end (and adding another line that will help with newer versions of npm)

### CODE REVIEW

**Comment by marybeshaw** [comment]
I have not tested this yet.  I can probably test it using the branch name in search-bifrost's .buildbacks file, when I try to deploy that to Node version 16, but I don't want to do that until after our midnight outage window on Sunday Night/Monday AM.

Maybe I can update the .buildpacks file without upgrading to node 16?

File: bin/compile [hunk 1]
```diff
@@ -1,62 +1,48 @@
...
+echo "       Organizations: ${ORGS}"
 
 ##
-# Set github config to always use https
-`git config --global url."https://".insteadOf git://`
\ No newline at end of file
+# Set GitHub config to always use https
+`git config --global url."https://".insteadOf git://`
+# This additional GitHub config will help for using HTTPS instead of SSH for npm versions 7+
+`git config --global url."https://github.com".insteadOf ssh://git@github.com`
```
Comment (marybeshaw) [inline bin/compile:48]: This is the line I added.  I found this issue in Travis (I'd already set it locally), and this extra line will help "ssh" type urls from a package-lock.json or package.json be coerced into "https" type urls.

**Comment by marybeshaw** [comment]
"sendToHeroku" step in CloudBees worked fine.
```
remote: =====> Downloading Buildpack: https://github.com/fs-webdev/heroku-buildpack-github-netrc.git#bed8db07ce8e16f1558d5bb0fb1a47042c741e40        
remote: =====> Detected Framework: github-netrc        
remote:        Generated .netrc & .curlrc files (available only at build-time)        
remote:        GitHub User:   fs-readonly        
remote:        Organizations: fs-webdev        
```
https://build.fsglobal.net/commander/link/workspaceFile/workspaces/default?jobStepId=831f833c-21bd-11ee-b224-0242eb1ba312&fileName=sendToHeroku.831f833c-21bd-11ee-b224-0242eb1ba312.log&jobName=frontier-search-bifrost-build-1624&jobId=5f0d8170-21bd-11ee-9dce-0242eb1ba312&diagCharEncoding=&resourceName=ecagent006&completed=1

**Review by redbugz — APPROVED** [review]
I don't have sufficient bash knowledge to know this is done correctly, but I trust whomever made this changes to the upstream repo to know what they are doing, what I see seems fine as long as it works

**Comment by marybeshaw** [comment]
A second team's build's "sendToHeroku" step with this branch!
https://github.com/fs-webdev/indexing/pull/5320/files
https://build.fsglobal.net/commander/link/jobDetails/jobs/f3418289-21c3-11ee-99af-0242eb1ba312

**Pull request merged**